node-api/handlers/debug/state.go

```diff
@@ -46,8 +47,8 @@ func (h *Handler[ContextT]) GetState(c ContextT) (any, error) {
 	}
 	return beacontypes.StateResponse{
 		// TODO: The version should be retrieved based on the slot
-		Version:             "deneb", // stubbed
```

Comment:
to avoid this, we should introduce:
- a `latest version`
- a version.String method

cc @calbera

Reply:
makes sense. `latest` version might not be necessary since we want to provide the version at which the incoming request is queried for. Would probably just require returning `chainSpec.ActiveForkVersionForSlot(requestedSlot).String()`